root/area/edit_form.tt

```diff
@@ -9,9 +9,27 @@
       [%- form_row_name_with_guesscase(r) -%]
       [%- form_row_text_long(r, 'comment', 'Disambiguation:') -%]
       [%- form_row_select(r, 'type_id', 'Type:') -%]
-      [%- form_row_text_list(r, 'iso_3166_1', 'ISO 3166-1:', 'ISO 3166-1') -%]
-      [%- form_row_text_list(r, 'iso_3166_2', 'ISO 3166-2:', 'ISO 3166-2') -%]
-      [%- form_row_text_list(r, 'iso_3166_3', 'ISO 3166-3:', 'ISO 3166-3') -%]
+      [% React.embed(c, 'static/scripts/edit/components/FormRowTextList', {
+           repeatable => form_to_json(form.field('iso_3166_1')),
+           label => 'ISO 3166-1:',
+           addButtonId => 'add-iso-3166-1',
+           addButtonLabel => 'Add ISO 3166-1',
+           removeButtonLabel => 'Remove ISO 3166-1',
```

Comment:
The labels in this file are still untranslated.

Reply:
They're intentionally untranslated, since this is an area editing page, so admin only. Did we have `l_admin` for TT?

Reply:
You're right, I overlooked that, sorry! We do have `l_admin` for TT in common-macros.tt so it probably makes sense to add that at least.